exporters/otlp/include/opentelemetry/exporters/otlp/otlp_environment.h

```diff
@@ -313,6 +315,50 @@ inline OtlpHeaders GetOtlpDefaultMetricHeaders()
   return result;
 }
 
+inline bool GetOtlpDefaultMetricsIsSslEnable()
+{
+  constexpr char kOtlpMetricsIsSslEnableEnv[] = "OTEL_EXPORTER_OTLP_METRICS_SSL_ENABLE";
+  constexpr char kOtlpIsSslEnableEnv[]        = "OTEL_EXPORTER_OTLP_SSL_ENABLE";
+
+  auto ssl_enable = opentelemetry::sdk::common::GetEnvironmentVariable(kOtlpMetricsIsSslEnableEnv);
+  if (ssl_enable.empty())
+  {
+    ssl_enable = opentelemetry::sdk::common::GetEnvironmentVariable(kOtlpIsSslEnableEnv);
+  }
+  if (ssl_enable == "True" || ssl_enable == "TRUE" || ssl_enable == "true" || ssl_enable == "1")
+  {
+    return true;
+  }
+  return false;
+}
+
+inline const std::string GetOtlpDefaultMetricsSslCertificatePath()
+{
+  constexpr char kOtlpMetricsSslCertificate[] = "OTEL_EXPORTER_OTLP_METRICS_CERTIFICATE";
+  constexpr char kOtlpSslCertificate[]        = "OTEL_EXPORTER_OTLP_CERTIFICATE ";
```

Comment:
1 more trailing space?

Reply:
oh bad. Thanks for noticing it, fixed now :(